realme_ota/utils/request.py

```diff
@@ -155,7 +157,8 @@ def set_body_headers(self):
         
         if self.req_version == 2:
             self.headers['version'] = '2'
-            
+            self.headers['mode'] = 'manual' # client_auto not work for Oneplus

```

Comment:
If this is specific to OnePlus, please relocate this under the previous model verification or implement an additional check at this point.

Reply:
updated.
still inside set_body_headers(), specific to OnePlus by checking self.properties.get('model')